docs/configuration/monitoring.md

```diff
@@ -12,32 +12,51 @@ we have all functionality available on Grafana.
 
 The simplest way to setup prometheus in your cluster is by using the
 [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
-bundle.
+bundle. Start by adding the custom resources required for installing this bundle in your cluster.
 
 ```sh
+# NOTE: if installing on microk8s use `microk8s.kubectl` instead of `kubectl`
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.36/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
+```
+
+Obtain the configurations files for [loki](monitoring-conf/loki-values.yaml),
+[promtail](monitoring-conf/promtail-values.yaml), and
+[prometheus-operator](monitoring-conf/prometheus-operator-values.yaml) and place them
+in the same folder where you'll run the helm commands.
+
+**You must edit:**
 
-sleep 5 # wait for crds to be created
+-   the Grafana password for the `admin` user in `prometheus-operator-values.yaml`
+-   the storage class for loki persistence in `loki-values.yaml`
 
-helm upgrade --install monitoring stable/prometheus-operator \
-  --version 6.9.3 \
-  --namespace monitoring \
-  --set prometheusOperator.createCustomResource=false \
-  --set grafana.service.type=NodePort\
-  --set grafana.adminPassword="CHANGE_HERE"
+You can now install the monitoring bundle on your cluster:
+
+```sh
```

Comment:
`sh` works just as fine as `bash`, but for the sake of consistency I suggest that we keep using "bash" throughout like in all other Markdown files in this repository.